classes/Validate.php

```diff
@@ -80,11 +80,19 @@ public static function isEmail($email)
             return false;
         }
 
+        $validator = Validation::createValidator();
```

Comment:
Just a question, in the BO there is a constraint regarding the length, shouldn't there be the same in FO?
Or maybe there is but not handled in this method?

Reply:
Handle in the form